src/index.html

```diff
@@ -21,7 +24,420 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header page__header">
+      <a
+        href="index.html"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo-img"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__links">
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="header__nav-item">
```